force-app/main/default/lwc/groupServiceDeliveries/groupServiceDeliveries.js

```diff
@@ -32,23 +42,40 @@ export default class GroupServiceDeliveries extends LightningElement {
         return this.currentStep && this.currentStep.value === 1;
     }
 
-    get isStep3() {
-        return this.currentStep && this.currentStep.value === 2;
-    }
-
     createSteps() {
         this._steps = new ProgressSteps();
         this._steps
-            .addStep("", "Selection")
-            .addStep("", "Service Selections", new NavigationItems().addNext())
-            .addStep("", "Select Contacts", new NavigationItems().addNext().addBack());
+            .addStep("", this.labels.serviceSelections, new NavigationItems().addNext())
+            .addStep(
+                "",
+                this.labels.selectContacts,
+                new NavigationItems().addNext().addBack()
+            )
+            .addStep("", this.labels.review);
     }
 
     handleOption1() {
-        // nav to step 2
+        this.createSteps();
+        this.currentStep = this._steps.currentStep;
+        this.isOptionSelection = false;
     }
 
     handleOption2() {
         this.dispatchEvent(new CustomEvent("hide"));
     }
+
+    handleNext() {
+        if (this.isStep2) {
```

Comment:
i think this is undefined as of now? probably ok though.

Reply:
The steps are defined once option 1 is selected. The screen just has words right now.